docs/wiki/handy-gcp-commands.md

```diff
@@ -29,57 +36,40 @@ SSH to GCP instance.
 
 Get into the docker container.
 ```
-docker exec -it $(docker ps -q) zsh
-cd /state/models
+docker exec -it elpis zsh
+cd /state/of_origin/models
 ls
 tar -cvf model.tar HASH_DIR_NAME
 ```
 
 Keep that Docker container running, and in another SSH terminal, copy from the container to the host.
 ```
-docker cp $(docker ps -q):/state/models/model.tar .
+docker cp elpis:/state/of_origin/models/model.tar .
 ```
 
 Copy from the host to the local machine (do this in a local terminal window).
 ```
-gcloud compute scp instance-3:~/model.tar ~/Downloads/model.tar
+gcloud compute scp instance-1:~/model.tar ~/Downloads/model.tar
 ```
 
 Otherwise, could share state dir from host into docker and save a few steps...
 
 
+## Fixing the SSH Key
 
-## Setting SSH key
+When making an SSH connection using gcloud, you may receive a `Remote Host changed` error. This can be fixed by regenerating some files on your computer.
 
-To fix `Remote Host changed` error, delete these files from your local machine.
+Use the Google Cloud Console in your browser to check that the VM is running.
+
+Delete these files from your computer.
 ```shell
 ~/.ssh/google_compute_engine
 ~/.ssh/google_compute_engine.pub
 ~/.ssh/google_compute_known_hosts
 ```
 
-Then recreate them. Start the VM in the browser interface. Login and generate new SSH keys.
+Run these commands on your computer to authorise you and generate new SSH keys. Replace the zone, instance and project names to suit your situation.
 ```shell
 gcloud auth login
 gcloud compute ssh --zone "us-central1-c" "instance-3"  --tunnel-through-iap --project "elpis-workshop"
```

Comment:
```suggestion
gcloud compute ssh --zone "us-central1-c" "instance-1"  --tunnel-through-iap --project "elpis-workshop"
```
Make consistent with other instance naming.

Reply:
thanks, changed in the next commit